.github/workflows/linux_meson.yml

```diff
@@ -102,3 +102,111 @@ jobs:
+      with:
+        submodules: recursive
+
+    - name: Setup Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Ubuntu dependencies
+      run: |
+        # NOTE: not the same OpenBLAS version as in upstream CI (I'm being lazy here)
+        sudo apt-get update
+        sudo apt-get install -y libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
+
+    - name: Caching Python dependencies
+      uses: actions/cache@v3
+      id: cache
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip
+
+    - name: Install Python packages
+      run: |
+        python -m pip install "setuptools<60.0" wheel ninja meson==0.60.3
+        python -m pip install --pre --upgrade --timeout=60 -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+        python -m pip install --pre --upgrade --timeout=60 cython pytest pytest-xdist pybind11 mpmath gmpy2 pythran threadpoolctl
+        python -m pip install matplotlib scikit-umfpack scikit-sparse
+        python -m pip uninstall -y scipy
+        python -m pip install pytest-cov coverage codecov gcovr
+
+    - name:  Prepare compiler cache
+      id:    prep-ccache
+      shell: bash
+      run: |
+        mkdir -p "${CCACHE_DIR}"
+        echo "::set-output name=dir::$CCACHE_DIR"
+        NOW=$(date -u +"%F-%T")
+        echo "::set-output name=timestamp::${NOW}"
+
+    - name: Setup compiler cache
+      uses:  actions/cache@v3
+      id:    cache-ccache
+      # Reference: https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#matching-a-cache-key
+      # NOTE: The caching strategy is modeled in a way that it will always have a unique cache key for each workflow run
+      # (even if the same workflow is run multiple times). The restore keys are not unique and for a partial match, they will
+      # return the most recently created cache entry, according to the GitHub Action Docs.
+      with:
+        path: ${{ steps.prep-ccache.outputs.dir }}
+        # Restores ccache from either a previous build on this branch or on main
+        key:  ${{ github.workflow }}-${{ matrix.python-version }}-ccache-linux-${{ steps.prep-ccache.outputs.timestamp }}
+        # This evaluates to `Linux Tests-3.9-ccache-linux-` which is not unique. As the CI matrix is expanded, this will
+        # need to be updated to be unique so that the cache is not restored from a different job altogether.
+        restore-keys: |
+          ${{ github.workflow }}-${{ matrix.python-version }}-ccache-linux-
+
+    - name: Setup blas 64
+      run: |
+        target=$(python tools/openblas_support.py)
+        echo "[openblas64_]" > site.cfg
+        echo "library_dirs = $target/lib" >> site.cfg
+        echo "include_dirs = $target/include" >> site.cfg
+        echo "runtime_library_dirs = $target/lib" >> site.cfg
```

Comment:
We already install `libopenblas-dev`. I don't think we need to do this anymore.

Reply:
This does not do anything. Meson does not use `site.cfg`. Nor is ILP64 BLAS supported yet. So how about:
1. change this code with the correct `PKG_CONFIG_PATH` invocation (see `windows.yml`)
2. comment out this code, and add a comment above it with a `FIXME` saying ILP64 is not yet supported, and should be re-enabled after the 1.9.0 release.

Reply:
I am not sure what to add for the path, can you make a suggestion?